index.html

```diff
@@ -30,10 +30,10 @@
 
 <h2>Development</h2>
 
-<p>Changes to the language are developed by way of a process which provides guidelines for evolving an addition from an idea to a fully specified feature, complete with acceptance tests and multiple implementations. There are four “maturity” stages. The TC39 committee must approve acceptance for each stage.
+<p>Changes to the language are developed by way of a process which provides guidelines for evolving an addition from an idea to a fully specified feature, complete with acceptance tests and multiple implementations. There are Strawman stage and four “maturity” stages. The TC39 committee must approve acceptance for each stage.
```

Comment:
This sounds a little awkward. Perhaps "there are five stages: a strawman stage, and 4 maturity stages."?